Tools/Scripts/filter-test-logs

```diff
@@ -94,18 +92,24 @@ def filter_webdriver_tests(log_file_name):
             if 'tests ran as expected' in line:
                 print_line = True
                 print(line, end='')
+    return print_line
 
 def main():
     args = parser()
+    log_file = os.path.abspath(f'{args.output}')
     if args.filter_type == 'jsc':
-        filter_jsc_tests(args.output)
+        printed_results = filter_jsc_tests(log_file)
```

Comment:
Nit: `printed_results` makes me think of a string of all the streamed logs. Can we rename to something like `did_print_logs` or even `logs_printed`?

Reply:
Sure.